tests/cli/test_config.py

```diff
@@ -48,6 +52,7 @@ def make_temp_condarc(value=None):
         if value:
             with open(temp_path, "w") as f:
                 f.write(value)
+        YamlRawParameter.make_raw_parameters_from_file.cache_clear()
```

Comment:
This is required since the test updates a config file and resets the context. In order to pick up the new changes to the file we must clear the cache.

Reply:
Good idea!